styles/style.scss

```diff
@@ -120,16 +124,52 @@ $font-path: '../../fonts';
 
 .teams-grid {
     display: grid;
-    grid-template-columns: 1fr 1fr 1fr;
-    column-gap: $spacing-lg;
+    grid-template-columns: 1fr;
+    column-gap: $spacing-2xl;
+    row-gap: $spacing-lg;
 
-    .mzp-c-card {
-        grid-column: span 3;
+    .c-card {
```

Comment:
**Suggestion (design):**
So I know the design changed to compensate for the heavier wordcount on these cards, but I still think we should align each box to be the same height, and have the names, company names, and Learn More sections to be aligned. 
I used Flexbox in this example to show you what I mean. It would be better for design to have things be nice and aligned. Although, you may have to set image dimensions in CSS to make sure they all stay consistent.

```
  display: flex;
  flex-direction: column;
  justify-content: space-between;
```

<img width="1241" alt="Screen Shot 2022-12-16 at 4 19 11 PM" src="https://user-images.githubusercontent.com/42309026/208190788-d454884a-0ab4-4835-bc5c-b101f91b024f.png">